Thirdweb.uplugin

```diff
@@ -1,25 +1,35 @@
 {
-    "FileVersion": 3,
-    "Version": 2,
-    "VersionName": "1.1.0",
-    "FriendlyName": "Thirdweb",
-    "Description": "Unreal Integration of Thirdweb Wallets",
-    "Category": "Thirdweb",
-    "CreatedBy": "Thirdweb",
-    "CreatedByURL": "https://thirdweb.com",
-    "DocsURL": "https://portal.thirdweb.com/unreal",
-    "MarketplaceURL": "com.epicgames.launcher://ue/marketplace/product/f21200c2610146f3888172994448e50d",
-    "SupportURL": "https://thirdweb.com/support",
-    "CanContainContent": true,
-    "IsBetaVersion": false,
-    "IsExperimentalVersion": false,
-    "Installed": false,
-    "Modules": [
-        {
-            "Name": "Thirdweb",
-            "Type": "Runtime",
-            "LoadingPhase": "Default",
-            "PlatformAllowList": ["Win64"]
-        }
-    ]
-}
+	"FileVersion": 3,
+	"Version": 3,
+	"VersionName": "1.1.1",
+	"FriendlyName": "Thirdweb",
+	"Description": "Unreal Integration of Thirdweb Wallets",
+	"Category": "Thirdweb",
+	"CreatedBy": "Thirdweb",
+	"CreatedByURL": "https://thirdweb.com",
+	"DocsURL": "https://portal.thirdweb.com/unreal",
+	"MarketplaceURL": "com.epicgames.launcher://ue/marketplace/product/f21200c2610146f3888172994448e50d",
+	"SupportURL": "https://thirdweb.com/support",
+	"CanContainContent": true,
+	"IsBetaVersion": false,
+	"IsExperimentalVersion": false,
+	"Installed": false,
+	"Modules": [
+		{
+			"Name": "Thirdweb",
+			"Type": "Runtime",
+			"LoadingPhase": "PreDefault",
+			"PlatformAllowList": [
+				"Win64",
+				"Mac",
+				"Linux"
```

Comment:
Should LinuxArm64 be added here?

Reply:
Also, should the mobile static libs be removed from the package for the time being?

Reply:
Needs testing then maybe